src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -394,6 +395,7 @@ struct SvgStyleProperty
     SvgColor color;
     bool curColorSet;
     char* cssClass;
+    bool paintOrder; //true if default (fill, stroke), false otherwise
```

Comment:
```isPaintOrderDefault``` ? a bit long, but more descriptive

Reply:
Remember this, "When in Rome, do as the Romans do"

But if you think it's necessary then please do it.

Reply:
moved to #1371